tests/nested/manual/uc20-fde-hooks/task.yaml

```diff
@@ -14,10 +14,6 @@ prepare: |
   echo "Build a kernel snap with the fde-setup hook"
   # shellcheck source=tests/lib/prepare.sh
   . "$TESTSLIB/prepare.sh"
-  # shellcheck source=tests/lib/nested.sh
-  . "$TESTSLIB/nested.sh"
-
-  mkdir "$(nested_get_extra_snaps_path)"
```

Comment:
Is this removal intended?

Reply:
hi, thanks for the review, yes it is intentional, the path is not being used so we don't need to create it.